main.go

```diff
@@ -15,11 +15,11 @@ import (
 	"github.com/SkynetLabs/skynet-accounts/email"
 	"github.com/SkynetLabs/skynet-accounts/jwt"
 	"github.com/SkynetLabs/skynet-accounts/metafetcher"
+	"github.com/joho/godotenv"
+	"github.com/stripe/stripe-go/v71"
 	"gitlab.com/SkynetLabs/skyd/skymodules"
 
-	"github.com/joho/godotenv"
 	"github.com/sirupsen/logrus"
-	"github.com/stripe/stripe-go/v71"
 	"gitlab.com/NebulousLabs/errors"
 )
 
```

Comment:
Should the env var names be `const` instead of `var`?

Reply:
Sure.